packages/rum-core/src/domain/assembly.ts

```diff
@@ -109,6 +109,7 @@ export function startRumAssembly(
             session: {
               plan: session.hasPremiumPlan ? RumSessionPlan.PREMIUM : RumSessionPlan.LITE,
             },
+            rule_psr: 'sampleRate' in configuration ? configuration.sampleRate : undefined,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:**  it should be `configuration.tracingSampleRate`, not `configuration.sampleRate`

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about putting this into [`computeRequestTracingInfo`](https://github.com/DataDog/browser-sdk/blob/d4c60b83bf09e890966383cca1b6c01336d9bb56/packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts#L130)?

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`tracingSampleRate` should always be defined on `RumConfiguration` values, so no need to check for its presence